server/app/genbank.py

```diff
@@ -102,3 +112,96 @@ def convert_gbk_to_vector(genbank_file, level: VectorLevel) -> GenbankData:
         features=features,
         references=references,
     )
+
+
+# Converts a model.Vector to a Genbank output
```

Comment:
Should be a docstring. Be a little more clear (Is it a levelN or any Vector? Why only LevelN? Are you serialising to GenBank format in a string or is the output a filename? etc)
* I would prefer a name like `serialise_to_genbank()`

Reply:
I've added the docstring and changed the name to "serialize_to_genbank".

Strictly speaking you could use this function for generating a genbank file of any kind of vector. I'm not sure if it makes sense to generate a genbank file of a level 0 vector  or a backbone vector, since the user probably already has it (when you submit a level 0 or backbone, you provide a genbank file).

From that perspective it made sense to use "levelN" in the function name...